hazelcast/src/test/java/com/hazelcast/jet/core/ExecutionLifecycleTest.java

```diff
@@ -882,6 +883,40 @@ public void when_processorInitBlocks_then_otherJobsNotBlocked() throws Exception
         TestProcessors.assertNoErrorsInProcessors();
     }
 
+    @Test
+    public void when_processorThrowsNonSerializable_thenItsWrapped() {
+        // Given
+        SupplierEx<ProcessorSupplier> supplier = PSThrowingNonSerializable::new;
+        DAG dag = new DAG().vertex(new Vertex("test", new MockPMS(supplier)));
+
+        // When
+        try {
+            Job job = newJob(dag);
+            job.join();
+            fail("Job execution should have failed");
+        } catch (Throwable e) {
+            assertContains(e.getMessage(), "bum!");
```

Comment:
```suggestion
            assertContains(e.getMessage(), "boom!");
```

Reply:
https://dictionary.cambridge.org/dictionary/english/bum

Reply:
I was Polishing the code :D